app/src/main/java/org/wikipedia/notifications/NotificationEditorTasksHandler.java

```diff
@@ -62,7 +63,8 @@ public static void dispatchEditorTaskResults(@NonNull Context context, @NonNull
     }
 
     public static void maybeShowEditDescriptionUnlockNotification(@NonNull Context context, boolean forced) {
-        if (!WikipediaApp.getInstance().isAnyActivityResumed() || forced) {
+        if (!WikipediaApp.getInstance().isAnyActivityResumed() && !Prefs.isSuggestedEditsAddDescriptionsMessageShown() || forced) {
+            Prefs.setSuggestedEditsAddDescriptionsMessageShown(true);
```

Comment:
I don't think I understand the purpose of this new preference. Do we need it? It looks like it prevents the notification (or dialog) from being shown more than once.  But what if the user logs out, and then another user logs in, not knowing about these notifications? Or is my understanding wrong?

Reply:
The idea is to prevent that if the **same user** logs out and logs in again, it should not see the dialog/notification again, and it is true that other user will not see the dialog/notification after passed the targets.

Reply:
Removed!